### ISSUE TITLE
Add polls for guests

### ISSUE THREAD

**Issue by mariebmo** [aslakhol/shera#99]
At times it'd be nice to have a poll for people attending - like having a poll for the movie to watch at a movie-night, or a theme for the upcoming quiz.

**Comment by aslakhol** [comment]
Yes, we certainly need some sort of system for easy questions. In addition to the sort of polls you're presenting, asking for food preferences when ordering is very useful

**Comment by aslakhol** [comment]
I think the primary use case here would maybe be scheduling. There are other good apps that do this (https://app.rallly.co/ is my current favourite), so I haven't prioritised it. However, I've been getting requests for this, and it would be handy to do something simple right in Shera. 

Open for thoughts!

**Comment by SindreKjelsrud** [comment]
Would it be possible to use something like [Ninjavote](https://github.com/mikkelsvartveit/ninjavote) for this through an iframe maybe? 
I don't know if that would be the most optimal way, or if it even aligns with Sheras vision, but just a thought😅
